helm/overstroomik/values.yaml

```diff
@@ -6,7 +6,7 @@ replicaCount: 1
 
 images:
   geoserver:
-    repository: deltares/overtroomik-geoserver
```

Comment:
@SiggyF You made a complete new repository! I've changed it here and deleted the image.